linux_os/guide/services/obsolete/tftp/tftp_uses_secure_mode_systemd/shared/stig/shared.yml

```diff
@@ -0,0 +1,34 @@
+srg_requirement: |-
```

Comment:
The `stig` directory should be in the `policy` directory, not in `shared` directory.